wemake_python_styleguide/visitors/tokenize/primitives.py

```diff
@@ -46,6 +49,13 @@ def _check_bad_number_suffixes(self, token: tokenize.TokenInfo) -> None:
                 BadNumberSuffixViolation(token, text=token.string),
             )
 
+    def _check_meaningless_zeros(self, token: tokenize.TokenInfo) -> None:
+        if any(token.string.lower().startswith(prefix) for prefix
```

Comment:
Can you please create a new var from this condition? It does not look good to me.